lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -33,8 +32,8 @@ local properties = {
 }
 
 local links = {
-  line_rgx = vim.regex([[\(\(^\|\s\+\)\[\[\)\@<=\(\*\|\#\|file:\)\?\(\(\w\|\/\|\.\|\\\|-\|_\|\d\)\+\)\?]]),
-  rgx = vim.regex([[\(\*\|\#\|file:\)\?\(\(\w\|\/\|\.\|\\\|-\|_\|\d\)\+\)\?$]]),
+  line_rgx = vim.regex([[\(\(^\|\s\+\)\[\[\)\@<=\(\*\|#\|file:\)\?\(\(\w\|\/\|\.\|\\\|-\)\+\)\?]]),
```

Comment:
Why was this regex changed?

Reply:
I was hunting a bug (for which I currently only have a workaround), which I suspect to be in the regexp. Because of that I took a deeper dive into the regexp documentationand found out, that it is not the minimal regexp for what it tries to match. Some escapes are unnecessary and the `\d` and underscore were redundant, because they are already captured by `\w`. I actually tried to explain both simplifications in the commit message.

Reply:
Thanks for the clarification!